components/02-elements/mega-menu/mega-menu.config.js

```diff
@@ -0,0 +1,153 @@
+module.exports = {
+  title: 'Mega menu',
+  collated: true,
```

Comment:
we don't have variants, so it's no necessary

Reply:
done.